Encryption.md

```diff
@@ -39,32 +39,32 @@ speed and access control granularity.
 2. Implement "client-side" encryption/decryption (storage client). The storage server 
 must not see plaintext data, metadata or encryption keys.
 3. Leverage authenticated encryption that allows clients to check integrity of the 
-retrieved data - making sure the file (or file parts) had not been replaced with a 
+retrieved data - making sure the file (or file parts) have not been replaced with a 
 wrong version, or tampered with otherwise.
 4. Support column access control - by enabling different encryption keys for different 
```

Comment:
As I noted above, this goal should be rephrased to state that columns can use different encryption keys. It should not claim to provide access control.